velox/core/QueryConfig.h

```diff
@@ -102,6 +111,16 @@ class QueryConfig {
     return get<uint64_t>(kMaxPartialAggregationMemory, kDefault);
   }
 
+  uint64_t maxExtendedPartialAggregationMemoryUsage() const {
+    static constexpr uint64_t kDefault = 1L << 24;
+    return get<uint64_t>(kMaxExtendedPartialAggregationMemory, kDefault);
+  }
+
+  double partialAggregationGoodPct() const {
+    static constexpr double kDefault = 0.5;
```

Comment:
Why are we increasing it to 0.5 now?

Reply:
0.5pct is 1/200 and let's start with that as default?

Reply:
Sounds good. Do you want to do more shadowing to understand better on this feature's effect on Deltoid queries?

Reply:
Yeah, will do. Thanks!